stac_fastapi/opensearch/stac_fastapi/opensearch/basic_auth.py

```diff
@@ -0,0 +1,115 @@
+"""Basic Authentication Module."""
+
```

Comment:
We should probably add this to the core module and then we can import it in es, os and mongo

Reply:
I'm not sure how to do this.

Reply:
The file would just go in the core folder - where core.py is - and then we can import it like from stac_fastapi.core.basic_auth import

Reply:
Nothing too fancy